File: NHSOnline.Backend.Worker.Mocking/Models/BodyEncoding.cs
namespace NHSOnline.Backend.Worker.Mocking.Models
{
    public class BodyEncoding
    {
        public BodyEncoding()
        {
            CodePage = 65001;
            EncodingName = "Unicode (UTF-8)";
            WebName = "utf-8";
        }

        public int CodePage { get; set; }
        public string EncodingName { get; set; }
        public string WebName { get; set; }
    }
}

File: bddtests/src/main/kotlin/mocking/models/BodyEncoding.kt
package mocking.models

class BodyEncoding {

    var codePage: Int = 0
    var encodingName: String? = null
    var webName: String? = null

    init {
        codePage = 65001
        encodingName = "Unicode (UTF-8)"
        webName = "utf-8"
    }
}